examples/chip-tool/commands/common/CredentialIssuerCommands.h

```diff
@@ -60,14 +60,15 @@ class CredentialIssuerCommands
      *
      * @param[in] nodeId   The desired NodeId for the generated NOC Chain - May be optional/unused in some implementations.
      * @param[in] fabricId The desired FabricId for the generated NOC Chain - May be optional/unused in some implementations.
+     * @param[in] cats     The desired CATs for the generated NOC Chain - May be optional/unused in some implementations.
      * @param[in] keypair  The desired Keypair for the generated NOC Chain - May be optional/unused in some implementations.
      * @param[inout] rcac  Buffer to hold the Root Certificate of the generated NOC Chain.
      * @param[inout] icac  Buffer to hold the Intermediate Certificate of the generated NOC Chain.
      * @param[inout] noc   Buffer to hold the Leaf Certificate of the generated NOC Chain.
      *
      * @return CHIP_ERROR CHIP_NO_ERROR on success, or corresponding error code.
      */
-    virtual CHIP_ERROR GenerateControllerNOCChain(chip::NodeId nodeId, chip::FabricId fabricId, chip::Crypto::P256Keypair & keypair,
-                                                  chip::MutableByteSpan & rcac, chip::MutableByteSpan & icac,
-                                                  chip::MutableByteSpan & noc) = 0;
+    virtual CHIP_ERROR GenerateControllerNOCChain(chip::NodeId nodeId, chip::FabricId fabricId, chip::CATValues cats,
```

Comment:
Should that be a `const CATValues & cats`, so we don't end up copying it as we go?

Reply:
done